src/components/asset-list/RecyclerAssetList2/core/useMemoBriefSectionData.ts

```diff
@@ -45,7 +46,7 @@ export default function useMemoBriefSectionData({
   const { isClaimablesOpen } = useOpenClaimables();
   const { isCoinListEdited } = useCoinListEdited();
   const hiddenAssets = useUserAssetsStore(state => state.hiddenAssets);
-  const { openFamilies } = useOpenFamilies();
+  const openCollections = useOpenCollectionsStore(state => state.openCollections);
```

Comment:
if this is used a bunch maybe nice to just have useOpenCollections() but not necessary